moped-database/hasura-cluster

```diff
@@ -707,6 +291,57 @@ function show_banner() {
   echo "- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -";
 }
 
+function __check_export_complete() {
+  local file_path="$1"
+  local last_line=$(tail -n 3 "$file_path" | head -n 1)
+  
+  if [[ "$last_line" == *"PostgreSQL database export complete"* ]]; then
+    # echo "PostgreSQL database export is complete."
+    return 0
+  else
+    # echo "PostgreSQL database export is not complete."
+    return 1
+  fi
+}
+
+function replicate() {
+  source ./env;
+  start_only;
+  # Get the current date in YYYY-MM-DD format
+  current_date=$(date +"%Y-%m-%d");
+  snapshot_file="./snapshots/production-$current_date.sql";
```

Comment:
This system should get a new file every day, but only once a day. Otherwise it will reuse the same snapshot all day unless you delete it to force it to get a new one.